dupcop

```diff
@@ -19,19 +20,18 @@ const getVariants = (string) => ({
 });
 
 const substitute = (string, substitutionsPairs) => {
-  for (let pair of substitutionsPairs) {
-    const what = getVariants(pair[0]);
-    const to = getVariants(pair[1]);
+  for (const pair of substitutionsPairs) {
+    const [ what, to ] = pair.map(getVariants);
     Object.keys(what).forEach(casing => {
-      const regexp = RegExp(what[casing], 'gm');
+      const regexp = new RegExp(what[casing], 'gm');
```

Comment:
Creating instances only with `[[Construct]]` is where modern JavaScript is going, so I would rather add `new`. And FWIW, it is easier to lint.